lms/djangoapps/ccx/plugins.py

```diff
@@ -31,7 +33,7 @@ def is_enabled(cls, course, user=None):
             # If ccx is not enable do not show ccx coach tab.
             return False
 
-        is_staff_or_instructor = has_access(user, 'staff', course) or has_access(user, 'instructor', course)
+        is_staff_or_instructor = user.has_perm(VIEW_CCX_COACH_DASHBOARD, course)
         if hasattr(course.id, 'ccx') and is_staff_or_instructor:
```

Comment:
I'd probably remove the `is_staff_or_instructor` variable (or rename it), since it's talking about the role not the permission.